webapp/src/nativeApp.ts

```diff
@@ -19,14 +19,10 @@ export function importNativeAppSettings() {
     NativeApp.settingsBlob = null
 }
 
+export function notifySettingsChanged(key: string) {
+    postWebKitMessage({type: 'didChangeUserSettings', key, settingsBlob: exportUserSettingsBlob()})
+}
+
 function postWebKitMessage(message: any) {
```

Comment:
Love this construct - for extending webapp-to-native messaging in the future. We'll add a mental todo to also adapt this for Windows / Linux in the future.